app/src/main/java/org/mozilla/fenix/settings/PairInstructionsFragment.kt

```diff
@@ -0,0 +1,58 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+package org.mozilla.fenix.settings
+
+import android.os.Bundle
+import android.view.LayoutInflater
+import android.view.View
+import android.view.ViewGroup
+import android.widget.Button
+import android.widget.TextView
+import androidx.appcompat.app.AppCompatActivity
+import androidx.core.text.HtmlCompat
+import androidx.navigation.Navigation
+import com.google.android.material.bottomsheet.BottomSheetDialogFragment
+import mozilla.components.support.base.feature.BackHandler
+import org.mozilla.fenix.R
+
+class PairInstructionsFragment : BottomSheetDialogFragment(), BackHandler {
+    override fun onCreate(savedInstanceState: Bundle?) {
+        super.onCreate(savedInstanceState)
+        (activity as AppCompatActivity).title = getString(R.string.preferences_sync)
+    }
+
+    override fun onCreateView(inflater: LayoutInflater, container: ViewGroup?, savedInstanceState: Bundle?): View? {
+        return inflater.inflate(R.layout.fragment_pair_instructions, container, false)
+    }
+
+    override fun onResume() {
+        super.onResume()
+        (activity as AppCompatActivity).title = getString(R.string.preferences_sync)
```

Comment:
We also usually add below this:
``(activity as AppCompatActivity).supportActionBar?.show()``